andchir/omnipay-bundle/1.0/config/packages/omnipay.yaml

```diff
@@ -0,0 +1,78 @@
+omnipay:
+    success_url: '/profile/history_orders'
+    fail_url: '/'
+    return_url: '/omnipay_return'
+    notify_url: '/omnipay_notify'
+    cancel_url: '/omnipay_cancel'
+    data_keys:
+        paymentId: ['orderNumber']
+        customerEmail: ['customerNumber']
+    gateways:
+        PayPal_Express:
+            parameters:
+                username: xxxxxxxxxxxxx
+                password: xxxxxxxxxxxxxxxxxx
+                signature: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+            purchase:
+                username: ~
```

Comment:
I would like you to remove all these default configuration.

Reply:
Sorry, I think it could save someone time. I do not see a problem to leave the default configuration.

Reply:
Because the recipe will work just as well without them.